vehicle/mkv/software/air_control/tests/test_initial_checks.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+import time
+from formula.projects.microhitl import Values as PinValue
+
+
+def reset(iocontroller, pins):
+    iocontroller.write_pin(pins["RESET"][0], PinValue.LOW)
+    time.sleep(0.5)
+    iocontroller.write_pin(pins["RESET"][0], PinValue.HIGH)
+
+
+def test_bms_can_timeout(canbus, iocontroller, pins):
+    """
+    Fault during init if BMS message takes longer than 1 second to receive
+    """
+    reset(iocontroller, pins)
+    assert canbus.get_state("air_fault") == "NONE"
```

Comment:
The `"air_fault"` here is, I'm guessing, the same string as what you changed your other `.yml` files to contain. You could consider making that string a constant somewhere in your Python code, or if the maintenance effort/workflow isn't too much, define it at an even higher level and template out both the `.yml` and `.py` files that reference it. (But that seems a little extreme, and I probably wouldn't recommend it.)